data/attributes.yml

```diff
@@ -30209,6 +30209,22 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3418
+  name: Dance style
+  description: Identifies the specific genre or type of dance
```

Comment:
This should include an example for consistent style guidance:

Indicates the specific genre or types of dance, e.g. flamenco, ballet